process/factory/indexerFactory.go

```diff
@@ -54,13 +54,6 @@ func NewIndexer(args ArgsIndexerFactory) (dataindexer.Indexer, error) {
 		return nil, err
 	}
 
-	dispatcher, err := dataindexer.NewDataDispatcher(args.IndexerCacheSize)
```

Comment:
can we also remove the args.IndexerCacheSize ?

Reply:
done